safep/fileIO.py

```diff
@@ -128,4 +128,136 @@ def read_Files(files, step=1):
     data.index = data.window
     data["dVdW"] = data.vdW_ldl - data.vdW_l
     
-    return data
\ No newline at end of file
+    return data
+
+
+def parse_Colvars_log(filename):
+    '''
+    Parses Colvars standard output from a log file
+
+    Returns dictionaries containing key-value pairs
+    global_conf: top-level Colvars parameters
+    colvars: list of dicts, one for each colvar
+    biases: list of dicts, one for each bias
+
+    Note: within colvars, parameters of sub-objects CVCs and atom groups are found as nested dictionaries
+    in a list under the 'children' key, e.g.:
+    colvars[0]['children'][0] is the first CVC of the first colvar
```

Comment:
This is representing the parse tree of Colvars, right? Can we call these 'branches'? or 'nodes'? 'children' makes me think of class inheritance. Maybe that's just me.

Reply:
They are called children in the Colvars source because they are dependents in the dependency resolution system. I can use another word but I need it to reflect that fact that this is a directed graph, which "nodes" doesn't. In graph theory, they'd be called "direct successors".
Eventually this function will be part of a `colvartools` Python module that doesn't exist just yet. So it may look slightly out of place here.